performance/README.md

```diff
@@ -0,0 +1,27 @@
+# How to run the fallout tests
+
+## Setup Graphite
+ 
+1. Create a new test based on the [graphite-setup.yaml](graphite-setup.yaml)
+2. Modify the parameters to match the scenario.
+    * Change the dse_version to match one of the versions you plan on testing against
+    * Change driver_oss_branch to be whatever branch of the driver you are using
+    * Driver_examples_branch should stay java-driver-4.x
+3. Start the graphite-setup (wait for the successful setup) and get the graphite server address:
+    * Navigate to running test, and obtain the ip from the observer node this will be used as a graphite server in the other tests. 
+    The ip is in the ctool-cluster-info.txt:
+    ![ctool-cluster-info](ctool-cluster-info.png)
+    * Login to the graphite server the address should match the observer’s, the web interface is on port 3000. 
+    The username/password is admin/admin
+    * Dashboards will be loaded automatically.
+    
+
+## Start performance tests      
+
+1. Create a new test based on the [oss-performance-setup.yaml](oss-performance-setup.yaml)
+2. Change the parameters in the same manner as the first test, with the addition of the graphite_host parameter. 
+Use the IP from the first cluster’s observer mode (the previous setup step).
```

Comment:
Not very clear. Also, you probably meant `node`, not `mode`?

Reply:
yes, `node`. changed